src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Object getRandomBall() {
+        Random random = new Random();
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        ball.setColor(randomColor);
+        ball.setNumber(random.nextInt(100));
+        return ball;
+    }

```

Comment:
It's not necessary to create a new instance of `Random` every time the method is called. Instead, you can define it as a private static final field in the class, which will be initialized only once when the class is loaded.